lib/transbank/sdk/patpass/patpass_by_webpay/base.rb

```diff
@@ -0,0 +1,46 @@
+module Transbank
+  module Patpass
+    module PatpassByWebpay
+      class Base
+
+        DEFAULT_API_KEY = '579B532A7440BB0C9079DED94D31EA1615BACEB56610332264630D42D0A36B1C'.freeze
+        DEFAULT_COMMERCE_CODE = '597055555550'.freeze
+        DEFAULT_MALL_COMMERCE_CODE = '597055555535'.freeze
```

Comment:
Is this correct?

Reply:
Yes! It's right! But the DEFAULT_MALL_COMMERCE_CODE is not defined in JIRA card https://transbank-developers.atlassian.net/browse/TD-323, this could be deleted

Reply:
I checked and "Patpass by Webpay Mall" is not on the JIRA card, and I think it doesn't even exist, so this constant shouldn't, either

Reply:
I've deleted that constant as you mentioned last before